go/cmd/dolt/dolt.go

```diff
@@ -144,32 +153,52 @@ func runMain() int {
 }
 
 func createMetricsEmitter(dEnv *env.DoltEnv) events.Emitter {
-	var emitter events.Emitter = events.NullEmitter{}
-	metricsDisabled := dEnv.Config.GetStringOrDefault(env.MetricsDisabled, "false")
+	var emitter events.Emitter = events.NewFileEmitter()
+	// var emitter events.Emitter = events.NullEmitter{}
 
-	if disabled, err := strconv.ParseBool(*metricsDisabled); err == nil && !disabled {
-		host := dEnv.Config.GetStringOrDefault(env.MetricsHost, env.DefaultMetricsHost)
-		portStr := dEnv.Config.GetStringOrDefault(env.MetricsPort, env.DefaultMetricsPort)
-		insecureStr := dEnv.Config.GetStringOrDefault(env.MetricsInsecure, "false")
+	// metricsDisabled := dEnv.Config.GetStringOrDefault(env.MetricsDisabled, "false")
 
-		port, err := strconv.ParseUint(*portStr, 10, 16)
+	// if disabled, err := strconv.ParseBool(*metricsDisabled); err == nil && !disabled {
+	// 	host := dEnv.Config.GetStringOrDefault(env.MetricsHost, env.DefaultMetricsHost)
+	// 	portStr := dEnv.Config.GetStringOrDefault(env.MetricsPort, env.DefaultMetricsPort)
+	// 	insecureStr := dEnv.Config.GetStringOrDefault(env.MetricsInsecure, "false")
 
-		if err != nil {
-			log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid port.", env.MetricsPort, *portStr))
-			return emitter
-		}
+	// 	port, err := strconv.ParseUint(*portStr, 10, 16)
 
-		insecure, err := strconv.ParseBool(*insecureStr)
+	// 	if err != nil {
+	// 		log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid port.", env.MetricsPort, *portStr))
+	// 		return emitter
+	// 	}
 
-		if err != nil {
-			log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid true/false value", env.MetricsInsecure, *insecureStr))
-		}
+	// 	insecure, err := strconv.ParseBool(*insecureStr)
 
-		hostAndPort := fmt.Sprintf("%s:%d", *host, port)
-		conn, _ := dEnv.GrpcConnWithCreds(hostAndPort, insecure, nil)
+	// 	if err != nil {
+	// 		log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid true/false value", env.MetricsInsecure, *insecureStr))
+	// 	}
 
-		return events.NewGrpcEmitter(conn)
-	}
+	// 	hostAndPort := fmt.Sprintf("%s:%d", *host, port)
+	// 	conn, _ := dEnv.GrpcConnWithCreds(hostAndPort, insecure, nil)
+
+	// 	return events.NewGrpcEmitter(conn)
+	// }
 
```

Comment:
Commented this code out just during development while working on the `FileEmitter`. 
It seems like this function is used to handle the creation of the default emitter we want, based on the various emitters we've created? 
Do we want the `FileEmitter` to now be the default emitter we instantiate?